fastrand 2.4.0 fails to compile on wasm32-unknown-unknown targets

Hello :)

I'm reporting what I assume to be a bug in fastrand, sorry in advance if this turns out to be bad configuration on my end

On wasm32-unknown-unknown target, with default features + "js", with fastrand v2.4.0, a `cargo check` reports:

```
    Checking fastrand v2.4.0
error[E0425]: cannot find function `getrandom` in crate `getrandom`
   --> /home/abadger/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/fastrand-2.4.0/src/global_rng.rs:223:16
    |
223 |     getrandom::getrandom(&mut seed).ok()?;
    |                ^^^^^^^^^ not found in `getrandom`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `fastrand` (lib) due to 1 previous error
```

Looking at the source of getrandom 0.3.4 which is what fastrand pulls in, looks like `getrandom::getrandom` was renamed to `getrandom::fill`?

Downgrading to fastrand 2.3.0 works for me, but it would be nice for this to be solved, and hopefully this info will help someone else at least.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -72,6 +72,9 @@ jobs:
       - name: Test wasm
         run: wasm-pack test --headless --chrome
         if: startsWith(matrix.os, 'ubuntu')
+      - name: Test wasm
+        run: wasm-pack test --headless --chrome --features js
+        if: startsWith(matrix.os, 'ubuntu')
       - run: cargo bench
         if: startsWith(matrix.rust, 'nightly')
 

--- a/src/global_rng.rs
+++ b/src/global_rng.rs
@@ -220,7 +220,7 @@ fn random_seed() -> Option<u64> {
 fn random_seed() -> Option<u64> {
     // TODO(notgull): Failures should be logged somewhere.
     let mut seed = [0u8; 8];
-    getrandom::getrandom(&mut seed).ok()?;
+    getrandom::fill(&mut seed).ok()?;
     Some(u64::from_ne_bytes(seed))
 }
 

